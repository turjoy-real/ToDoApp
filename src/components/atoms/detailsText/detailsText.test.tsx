import {render} from '@testing-library/react-native';
import DetailsText from './detailsText';

const mockText = 'test in jest with test ideas';

describe('Details Text Component', () => {
  it('renders text corectlty', () => {
    const {getByText} = render(
      <DetailsText item={{pending: true, description: mockText}} lines={1} />,
    );
    expect(getByText(mockText)).toBeTruthy();
  });

  it('matches snapshot', () => {
    const {toJSON} = render(
      <DetailsText item={{pending: true, description: mockText}} lines={1} />,
    );
    expect(toJSON()).toMatchSnapshot();
  });
});
