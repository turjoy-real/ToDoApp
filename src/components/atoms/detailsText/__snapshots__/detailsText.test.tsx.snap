// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Details Text Component matches snapshot 1`] = `
<Text
  numberOfLines={1}
  style={
    [
      {
        "textAlign": "left",
      },
      {
        "color": "rgba(28, 27, 31, 1)",
        "writingDirection": "ltr",
      },
      [
        {
          "fontFamily": "System",
          "fontSize": 24,
          "fontWeight": "400",
          "letterSpacing": 0,
          "lineHeight": 32,
        },
        {
          "color": "#dadada",
          "textAlign": "left",
          "textDecorationLine": "none",
        },
      ],
    ]
  }
>
  test in jest with test ideas
</Text>
`;
